src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -881,6 +888,19 @@ private static String processPageSourceForGodMode(String content) {
                 // jQuery-ui CDN
                 .replace("https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js");
     }
+    
+    private static String processPageSourceForGodModePartTwo(String content) {
```

Comment:
I believe this function is only called when we are generating expected htmls. 
So we could name this one as "processPageSourceForCreatingExpectedHtml()" or smth along those lines